test/e2e_source/kafka_binding_test.go

```diff
@@ -0,0 +1,30 @@
+//go:build e2e && source
+// +build e2e,source
+
+/*
+ * Copyright 2023 The Knative Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package e2e_source
+
+import (
+	"testing"
+
+	"knative.dev/eventing-kafka-broker/test/e2e_source/helpers"
+)
+
+func TestKafkaBinding(t *testing.T) {
+	helpers.TestKafkaBinding(t)
+}
```

Comment:
We don't support KafkaBinding in this repository

Reply:
Ah ok :+1: